test/postman-tests/resources/collections/Test_Runtime_API_Operations-collection.json

```diff
@@ -447,6 +447,71 @@
 			},
 			"response": []
 		},
+		{
+			"name": "RuntimeAPI GET API Definition",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"  pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Assert token generation response\", () => {",
+							"    //parse the response JSON and test three properties",
+							"    const responseJson = pm.response.json();",
+							"    pm.expect(responseJson.swagger.json).not.eql(undefined);",
+							"    pm.collectionVariables.set('APITestKey', responseJson.apikey);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"auth": {
+					"type": "bearer",
+					"bearer": [
+						{
+							"key": "token",
+							"value": "{{access_token}}",
+							"type": "string"
+						}
+					]
+				},
+				"method": "GET",
+				"header": [
+					{
+						"key": "Host",
+						"value": "api.am.wso2.com",
+						"type": "default"
+					}
+				],
+				"url": {
+					"raw": "{{gatewayBaseURl}}/api/runtime/1.0.0/swagger.json",
```

Comment:
is this path correct? this test also seems to fail with 503 service not available

Reply:
This test is removed. Will add this in a separate PR. Also, these are the failing tests.
``` TestApiRateLimiting:1 | expected response to have status code 200 but got 503
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-header-value'
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-header-value'
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-Response-header-value'
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-Response-header-value'
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-header-value'
TestRequestInterceptor:1 | expected undefined to equal 'Interceptor-header-value'
TestRequestInterceptor:1 | expected response to have status code 200 but got 429```

I ran rate limit tests locally and all were passed without any error. Regarding `Interceptor-header-value` issue, this occurs in this PR as well. https://github.com/wso2/apk/actions/runs/5199491115/jobs/9377265813?pr=1227